ApplicationService/transactioncontext.py

```diff
@@ -0,0 +1,9 @@
+class transactioncontext:
+    def __enter__(self):
+        raise NotImplementedError
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        raise NotImplementedError
+
+    def get_errors(self):
```

Comment:
can you return that List of BusinnessError, right?

Reply:
correctly